### PR TITLE
New Clubs Pages!

### PR DESCRIPTION
**Pull request by flukeout**
New clubs pages based on the work in https://github.com/mozilla/learning.mozilla.org/issues/2013

@gvn Looks like I may have bungled something up with my rebasing, there are a few commits here that shouldn't be. I'll ping you for help.

### CODE REVIEW

**Comment by flukeout** [comment]
Will retry

**Comment by alanmoo** [comment]
Hey @flukeout - just stumbled across this and I'm sure @gvn will note it too, but can you run the SVGs through https://www.npmjs.com/package/svgo? There's some extra data in them that tool will clean up.

**Comment by gvn** [comment]
Needs merge conflicts resolved...

File: less/pages/clubs.less [hunk 1]
```diff
@@ -1,4 +1,27 @@
...
+
+    .top-level-item.sub-nav-active {
+      background: #2e5289;
+
+      .sidebar-subitems {
+        background-color: #2e5289;
+      }
+
+      a {
+        background: transparent !important;
```
Comment (gvn) [inline less/pages/clubs.less]: Any way we can do this without `!important`?

File: less/mofo-ui-overrides.less [hunk 2]
```diff
@@ -0,0 +1,84 @@
+.mui-tab-switcher {
+  padding-bottom: 50px;
+  position: relative;
+  padding-top: 30px;
+
+  .panel {
+    box-shadow: none;
+  }
+
+  button.btn, button.btn:active, button.btn:hover {
```
Comment (gvn) [inline less/mofo-ui-overrides.less]: Generally we should avoid naming elements in selectors. Can this just be `.btn, .btn:active, .btn:hover`?

**Comment by flukeout** [comment]
O M G - it can merge. Will address nits when you're done reviewing @gvn

**Comment by gvn** [comment]
So, that squashed down to 1 commit, which says it's for localization, so just be sure to rename the commit to something meaningful when you merge...

**Comment by gvn** [comment]
Also, like @alanmoo mentioned, would be good to use `svgo` to optimize the SVG files.

File: lib/routes.jsx [hunk 3]
```diff
@@ -36,9 +36,9 @@ var pages = {
   'activities/web-lit-basics-two': require('../pages/web-lit-basics-two.jsx'),
   'activities/webmaker': require('../pages/webmaker.jsx'),
   'activities/back-to-school-write-the-web': require('../pages/back-to-school-write-the-web.jsx'),
-  'clubs': require('../pages/clubs.jsx'),
+  'clubs': require('../pages/clubs/index.jsx'),
   'clubs/list': require('../pages/clubs-list.jsx'),
-  'clubs/guides': require('../pages/clubs-guides/ClubsGuides.jsx'),
+  // 'clubs/guides': require('../pages/clubs-guides/ClubsGuides.jsx'),
```
Comment (gvn) [inline lib/routes.jsx]: Can just delete this line instead of commenting it out...

**Comment by flukeout** [comment]
@gvn fixed the nits and changed the commit message.

**Comment by flukeout** [comment]
Well, I didn't fix the commit message apparently.

**Comment by gvn** [comment]
hi5

R+!

**Comment by flukeout** [comment]
Merging as discussed with @gvn - will tackle svgo optimization separately.

**Comment by Pomax** [comment]
I can't believe this finally worked.

**Pull request merged**